packages/block-editor/src/components/block-switcher/index.js

```diff
@@ -138,6 +138,7 @@ export class BlockSwitcher extends Component {
 								label={ label }
 								onKeyDown={ openOnArrowDown }
 								showTooltip
+								tooltipPosition="bottom"
```

Comment:
Was this intended? I'm not sure it works well in all situations (mobile, top toolbar, full-width blocks...)

Reply:
Ah, I forgot about this... I added it because at one point the tooltip wasn't showing without it. Now it seems that the tooltip doesn't show regardless. There some conflict being caused by the `<Button />` component in the `block-parent-selector` and I'm unsure how to go about fixing this...